application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -58,7 +59,7 @@ private ModerationUtils() {
     static EmbedBuilder getModActionEmbed(Guild guild, String action, String description,
             String reason, boolean isPunishAction) {
         return new EmbedBuilder().setAuthor(guild.getName(), null, guild.getIconUrl())
-            .setTitle(action.toUpperCase())
+            .setTitle(action.substring(0, 1).toUpperCase() + action.substring(1))
```

Comment:
The design on this `action` seems incorrect.
1. u called it `action`, but what is it exactly? its supposed to be the **title**. but u give it the `ACTION_VERB`
2. the action verb is sth like "banned", "warned" - a verb. its not a title. You want it to say "Ban" and "Mute", not "Banned".

You should adjust the callsite and definitely improve the naming on your variable here, it should be `actionTitle` and in the javadoc it should give an example, like `"ban"`.